src/components/InputTypes/TextArea.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextArea = ({
+  input,
+  label,
+  disabled,
+  meta: { touched, error },
+}) => (
+  <div>
+    <label htmlFor="value">
```

Comment:
```suggestion
    <label>
```

The `for` is redundant.